index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Javascript-module2 content</title>
+</head>
+<body>
+	<button style="width: 200px; height: 40px;"><a href="./colour-changing-grid/index.html">colour changing grid</a></button>
+	<button style="width: 200px; height: 40px;"><a href="./tictactoe-game/index.html">tic tac toe game</a></button>

```

Comment:
I'm not a huge fan of using inline styles for markup. Using a class here would be best. In fact, you could get rid of the button altogether and style the `a` instead if you wanted to get fancy.

That's one of the benefits of having a custom button class, you could use it on both `button` and `a`.